oras.yaml

```diff
@@ -0,0 +1,40 @@
+package:
+  name: oras
+  version: 0.16.0
+  epoch: 0
+  description: OCI registry client - managing content like artifacts, images, packages.
+  target-architecture:
+    - all
+  copyright:
+    - license: Apache-2.0
+      paths:
+        - "*"
+  dependencies:
+    runtime:
+      - ca-certificates-bundle
+environment:
+  contents:
+    packages:
+      - ca-certificates-bundle
+      - busybox
+      - go
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/oras-project/oras
+      branch: v${{package.version}}
+      destination: oras
+  - runs: |
+      cd oras
+      make build-linux-amd64
+      make build-linux-arm64

```

Comment:
hmm?  why not use subpipelines here?

e.g.

```yaml
- assertions:
    required-steps: 1
  pipeline:
    - if: ${{build.arch}} == 'x86_64'
      runs: |
        make build-linux-amd64
    - if: ${{build.arch}} == 'aarch64'
      runs: |
        make build-linux-arm64
- runs: |
    install -m755 -D ./bin/linux/*/oras "${{targets.destdir}}"/usr/bin/oras
```

Reply:
Nice. Let me change it.

Reply:
cool!